implementations/app-swift/src/__tests__/cases/000-sanity/output/Types.swift

```diff
@@ -0,0 +1,243 @@
+    var else: String
+}
+
+
+// Objects END //
+
+// Enums START //
+
+public enum CustomEnum: String, Codable {
+    case STRING
+    case BYTES
+}
+
+public enum While: String, Codable {
+    case _for
+    case _in
+}
+
+
+// Enums END //
+
+// Imported objects START //
+
+public struct TestImportObject: Codable {
+    var object: TestImportAnotherObject
+    var optObject: TestImportAnotherObject?
+    var objectArray: Array<TestImportAnotherObject>
+    var optObjectArray: Array<TestImportAnotherObject?>?
+    var en: TestImportEnum
+    var optEnum: TestImportEnum?
+    var enumArray: Array<TestImportEnum>
+    var optEnumArray: Array<TestImportEnum?>?
+}
+
+public struct TestImportAnotherObject: Codable {
+    var prop: String
+}
+
+
+// Imported objects END //
+
+// Imported envs START //
+
+public struct TestImportEnv: Codable {
+    var object: TestImportAnotherObject
+    var optObject: TestImportAnotherObject?
+    var objectArray: Array<TestImportAnotherObject>
+    var optObjectArray: Array<TestImportAnotherObject?>?
+    var en: TestImportEnum
+    var optEnum: TestImportEnum?
+    var enumArray: Array<TestImportEnum>
+    var optEnumArray: Array<TestImportEnum?>?
+}
+
+// Imported envs END //
+
+// Imported enums START //
+
+public enum TestImportEnum: String, Codable {
+    case STRING
+    case BYTES
+}
+
+public enum TestImportEnumReturn: String, Codable {
+    case STRING
+    case BYTES
+}
+
+
+// Imported enums END //
+
+// Imported modules START //
+
+// URI: "testimport.uri.eth" //
+public struct TestImportArgsImportedMethod: Codable {
+    var str: String
+    var optStr: String?
+    var u: UInt32
+    var optU: UInt32?
+    var uArrayArray: Array<Array<UInt32?>?>
+    var object: TestImportObject
+    var optObject: TestImportObject?
+    var objectArray: Array<TestImportObject>
+    var optObjectArray: Array<TestImportObject?>?
+    var en: TestImportEnum
+    var optEnum: TestImportEnum?
+    var enumArray: Array<TestImportEnum>
+    var optEnumArray: Array<TestImportEnum?>?
+}
+
+// URI: "testimport.uri.eth" //
+public struct TestImportArgsAnotherMethod: Codable {
+    var arg: Array<String>
+}
+
+// URI: "testimport.uri.eth" //
+public struct TestImportArgsReturnsArrayOfEnums: Codable {
+    var arg: String
+}
+
+/* URI: "testimport.uri.eth" */
+class TestImport {
+    var client: PolywrapClient? = nil
+    var env: TestImportEnv? = nil
+    var uri: Uri? = nil
+
+    init(client: PolywrapClient? = nil, env: TestImportEnv? = nil, uri: Uri? = nil) {
+        self.client = client
+        self.env = env
+        self.uri = uri
+    }
+
+    func getDefaultClient() -> PolywrapClient {
+        if (self.client == nil) {
+            self.client = BuilderConfig().addSystemDefault().addWeb3Default().build()
+        }
+        return self.client!
+    }
+
+    func getDefaultUri() -> Uri {
+        if (self.uri == nil) {
+            self.uri = try? Uri("testimport.uri.eth")
+        }
+        return self.uri!
```

Comment:
since we are doing a `try?` here, handling this without `!` would mean to add  `throws` keyword in the definition of the function; hence, changing how the dev exp is... So let's keep it like this for now

noob q: this `Uri` should **never** be wrong the CLI would not create these bindings right? :thinking:

Reply:
I updated the pattern to match the `getDefaultClient` pattern. Let me know your thoughts!